custom/icds_reports/ucr/reports/mobile_mpr_6b_pse_attendance_per_year.json

```diff
@@ -108,6 +103,16 @@
         "field": "month",
         "format": "%Y-%m"
       },
+      {
+        "display": "Age Group",
+        "column_id": "age_group",
+        "type": "conditional_aggregation",
+        "whens": {
+            "36 <= age_in_months AND age_in_months < 48": "3_years",
+            "48 <= age_in_months AND age_in_months < 60": "4_years",
+            "60 <= age_in_months AND age_in_months <= 72": "5_years",
```

Comment:
do you know why the last condition on all of these is <= instead of < ? It seems inconsistent to me and could get questioned later